.github/workflows/coverage.yml

```diff
@@ -0,0 +1,42 @@
+name: build
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  push:
+    branches:
+      - 'main'
+      - 'release/*'
+    tags:
+      - 'v*'
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        go:
+          - 1.20
+          - 1.21
```

Comment:
Need to quote these, or use `1.20.x`, `1.21.x`, because YAML.. and it's now using `go1.2` 😂

Reply:
!